bin/ci

```diff
@@ -9,20 +9,28 @@ def main(options)
   hetzner_server_st = Prog::Test::HetznerServer.assemble(vm_host_id: options[:vm_host_id])
   wait_until(hetzner_server_st, "wait")
 
+  tests_to_wait = []
   if options[:test_cases].include?("vm")
     encrypted_vms_st = Prog::Test::VmGroup.assemble(storage_encrypted: true, test_reboot: true)
     log(encrypted_vms_st, "storage_encrypted: true")
-    wait_until(encrypted_vms_st)
+    tests_to_wait << [encrypted_vms_st]
 
     unencrypted_vms_st = Prog::Test::VmGroup.assemble(storage_encrypted: false, test_reboot: false)
     log(unencrypted_vms_st, "storage_encrypted: false")
-    wait_until(unencrypted_vms_st)
+    tests_to_wait << [unencrypted_vms_st]
   end
 
   github_runner_test_cases = options[:test_cases].select { _1.include?("github_runner") }
   if github_runner_test_cases
     runner_st = Prog::Test::GithubRunner.assemble(options[:vm_host_id], github_runner_test_cases)
-    wait_until(runner_st)
+    tests_to_wait << [runner_st]
+  end
+
+  # Although wait_until will be blocked while checking the first one
+  # it won't affect the total time as other strands will continue in parallel.
+  # No need to make it parallel.
+  tests_to_wait.map do |strand, label|
```

Comment:
I'm confused here. I thought you wanted to add a feature that waits for different labels for various tests. However, right now, all of them are set to wait until they exit. If you plan to retain this feature for future use, it could be beneficial to explicitly push `nil` for exiting tests, like `tests_to_wait << [runner_st, nil]`. Currently, no one else knows about this feature.

Reply:
I have passed it as `tests_to_wait << [unencrypted_vms_st]` since the label would be nil if there is no label passed while adding tests to tests_to_wait. Let me add nil for being more explicit.